syntax = "proto3";
import "DataFormat.proto";

message SetDataFormat {
	DataFormat newformat = 1;
}
